gensim/models/word2vec.py

```diff
@@ -387,6 +409,12 @@ def __init__(
 
         `iter` = number of iterations (epochs) over the corpus.
 
+        `trim_rule` = vocabulary trimming rule, specifies whether certain words should remain
+         in the vocabulary, be trimmed away, or handled using the default (discard if word count < min_count).
+         Must be a callable that accepts a tuple (word, count, min_count) and returns
```

Comment:
It doesn't have to be a callable; in fact, the doc doesn't describe the most common option, which is leaving it at `None` (optional param).